### PR TITLE
Corrected Cheese and formatting.md

### PR DESCRIPTION
**Pull request by Konkleffke**
Corrected Cheese duplicate and formatting.

## 🎯 Objective

Ensuring the formatting looks good. So its easy to understand for everyone using it. 
Corrected a minor mistake.

## 🛠️ Changes made

Edited the formatting into a nicer looking way. Removed cheese 🧀 

## 👥 Who needs to review this Pull Request? 

@anya-clowers @Juanhuahua

### CODE REVIEW

File: dinner/readme.md [hunk 1]
```diff
@@ -4,13 +4,18 @@
...
 
-## Equipment needed: Pizza Stone, Oven, Bowls, Taste Buds
+## Equipment needed: 
+Pizza Stone, Oven, Bowls, Taste Buds
 
 
-## Steps for preparation: 1) Make the Dough 2) Put the suace on the dough 3) Put cheese on the dough 4) Put it in the oven
+## Steps for preparation: 
+1) Make the Dough
+2)   Put the suace on the dough
```
Comment (Juanhuahua) [inline dinner/readme.md:17]: here can adjust a bit

Comment (sanvelappan) [inline dinner/readme.md:17]: there seems to be an extra space here for no reason. Why would you do that 🥇

**Review by sanvelappan — COMMENTED** [review]
please check the formatting before approving

File: dinner/readme.md (hunk 1 above)
Comment (yining-yan) [reply dinner/readme.md:17]: sharp eyes!

Comment (Konkleffke) [reply dinner/readme.md:17]: True i have to fix that.

**Pull request closed without merging**